README.md

```diff
@@ -1,143 +1,105 @@
+1. Test Your Change
+1. Commit Your Change
+1. (Optional) Share How This Change Looks like to Others
+1. Publish Your Change to Your Github Account
+1. Create a Merge Request and Participate the Review
 
-### fork the source codes
+### Fork the Repository
 
-Firstly, let's get the source codes. We will "copy" this source codes to your github account. The term "fork" means to get a derived copy which contents are the same as the source.
+First, you need to get the source of the website by "forking" it. That is, to get a derived copy which share the history of the source repository.
 
-Click **fork** on the up-right of this [cesehub](https://github.com/cesehub/cesehub/) github repository page to fork the source codes to your github account. If you have not github account, please regist one for you first.
+Click **Fork** button on the top-right of this [cesehub](https://github.com/cesehub/cesehub/) github page to fork it to your github account. (You need a github account ready.)
 
-### clone the source codes to your local machine
+### Clone the Forked Repository to Your Local Machine
 
-Clone the source codes to your local machine, so you could edit the codes on your laptop or desktop.
-In this session, we will get two working copies. One is a folder with the name **cesehub** and the other is **cesehub-dev**.
+You have to clone the forked repository to your local machine before you can edit the source code of the website. In the example of this section, we will get two working copies. One is a folder with the name `cesehub` and the other is `cesehub-dev`.
 
-On your github page holding your cesehub fork which usually looks like https://github.com/<your account name>/cesehub/, there is a button **Clone or download**. Click it and copy the url of **Clone with HTTPS** to get your-repo-url which will be used later.
+The URL of your cesehub repository on github should look like `https://github.com/<your account name>/cesehub/`. There is a button **Clone or download**. Click it and copy the URL of **Clone with HTTPS** to get `<your-repo-url>` which will be used later.
 
-Please open your termial application of your machine, and type in the following command.
+Please open your termial application of your machine, and type in the following command:
 
 ```
 git clone <your-repo-url>
 ```
-you-repo-url is the url you just got on your github page. Your system needs to install git first to use this application.
-Now you should have a folder named **cesehub** on your local machine.
-This repository will be used to host your fork github page. You could share the url it hosts to the others for their reviews. We will talk about the url later.
-We called this is a staging site.
-
-Finally, clone the source code directly from the cesehub upstream
-```
-git clone https://github.com/cesehub/cesehub/ cesehub-dev
-```
-
-You should have two folders on your local machine now. One is cesehub and another is cesehub-dev.
 
-##### remove CNAME
```

Comment:
Settling with a single clone looks much better.  Being able to use gh-pages branch to host the staging site is convenient.  I think it's worth of the efforts to explain the flow.

The README document can be improved.  But it can be done later.